gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -441,8 +463,14 @@ private int readObjectContentFromGCS(ByteBuffer byteBuffer) throws IOException {
       positionInGrpcStream += bytesToWrite;
 
       if (responseSizeLargerThanRemainingBuffer) {
+        invalidateBufferedContent();
         bufferedContent = content;
         bufferedContentReadOffset = bytesToWrite;
+        streamForBufferedContent = stream;
```

Comment:
I don't see any usage of this assignment to `streamForBufferedContent`, am I missing something here ?

Reply:
This is to keep the stream alive because deserialized messages point the buffer of the stream. Therefore stream should live as long as the messages can be accessible.

Reply:
Can you add this as a comment for future reference ?

Reply:
It already has those comments at the definition of this variable, though. But I can add more context here.